upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.12.yaml.template

```
@@ -121,6 +121,9 @@ data:
   install-iptables-rules: "{{- if .IPTablesRulesNoinstall -}}false{{- else -}}true{{- end -}}"
   auto-direct-node-routes: "{{- if .AutoDirectNodeRoutes -}}true{{- else -}}false{{- end -}}"
   enable-node-port: "{{- if .EnableNodePort -}}true{{- else -}}false{{- end -}}"
+  {{ if .EnableNodePort }}
+  kube-proxy-replacement: strict
+  {{ end }}
```

Should be easier to deal with the extra white space:
```suggestion
  kube-proxy-replacement: "{{- if .EnableNodePort -}}strict{{- else -}}probe{{- end -}}"
```